router/pkg/config/config.schema.json

```diff
@@ -851,6 +851,11 @@
                   "default": false,
                   "description": "Enable the collection of metrics for the GraphQL operation router caches. The default value is false."
                 },
+                "engine_stats" : {
```

Comment:
We might want to add more in the future. I'd try to be more specific e.g.

```yaml
engine_stats
  connections: true
  websockets: true
```

This also allow the user to opt-in to specific metrics without buying into all metrics which is always a cost factor for them.